packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -272,6 +272,12 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
     );
   }
 
+  if (wco.tsConfig.options.target === 4) {
+    wco.logger.warn(`\nWARNING: Zone.js does not support native async/await
```

Comment:
Hi, you need to use `stripIndent` in order to avoid having extra spacing in the terminal.

```
        wco.logger.warn(tags.stripIndent`
          WARNING: Zone.js does not support native async/await in ES2017.  
          These blocks are not intercepted by zone.js and will not triggering change detection.
          See: https://github.com/angular/zone.js/pull/1140 for more information.
        `);
      
```